src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountExtractPdpEntryServiceImpl.java

```diff
@@ -207,25 +207,26 @@ private void addOnlyPositiveAccountingEntryForPDPProcessing(PdpFeedDetailEntry n
         if (originalAccountingEntry.getAmount().isPositive()) {
             newDetailEntry.getAccounting().add(copyAccountingEntry(originalAccountingEntry));
         } else {
-            LOG.debug("addOnlyPositiveAccountingEntryForPDPProcessing, not adding accounting entry: " + originalAccountingEntry.toString());
+            LOG.info("addOnlyPositiveAccountingEntryForPDPProcessing, not adding accounting entry: " + originalAccountingEntry.toString());
         }
     }
 
     private void addNewPaymentDetailToGroupIfTotalIsPositive(PdpFeedGroupEntry newGroupEntry, PdpFeedDetailEntry newDetailEntry, KualiDecimal originalDetailTransactionTotal) {
-        if (LOG.isDebugEnabled()) {
-            LOG.debug("addNewPaymentDetailToGroupIfTotalIsPositive, total transaction for " + newDetailEntry.getSourceDocNbr() + " detail: " + originalDetailTransactionTotal);
-        }
         if (originalDetailTransactionTotal.isPositive()) {
+            if (LOG.isDebugEnabled()) {
```

Comment:
That was intentional.  It was intentional, but I am hemming and hawing on it.  Erroring on the side of more logging is better at first, I will switch it.